demos/button.html

```diff
@@ -47,9 +52,19 @@
       fieldset legend {
         display: block;
         padding: 16px;
-        padding-top: 64px;
+        padding-top: 48px;
         padding-bottom: 24px;
       }
+      h1 {
+        padding-left: 36px;
+        padding-top: 64px;
+        padding-bottom: 8px;
```

Comment:
Nit: I think this looks fine without the 8px bottom padding, since there's already plenty of bottom margin coming from normalize.  Not sure if you had a specific reason for adding this.